### PR TITLE
Satellite scalling

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

Add option to scale satellite image by max and min values, theres also an option to not do any scaling

Helps  #361 

## How Has This Been Tested?

Ran locally with india-forecast-app, and values go down from ~1000, ~<1 values

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: ocf_datapipes/training/pvnet_site.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
     NWP_MEANS,
     NWP_STDS,
     RSS_MEAN,
+    RSS_RAW_MAX,
+    RSS_RAW_MIN,
     RSS_STD,
```
Comment (AUdaltsova) [inline ocf_datapipes/training/pvnet_site.py:29]: Sorry to be like this, but shouldn't these be in grouped? min-max together and mean-std together

File: ocf_datapipes/transform/xarray/normalize.py [hunk 2]
```diff
@@ -22,6 +22,8 @@ def __init__(
         max_value: Optional[Union[int, float]] = None,
         calculate_mean_std_from_example: bool = False,
         normalize_fn: Optional[Callable] = None,
+        min_values: Optional[np.ndarray] = None,
+        max_values: Optional[np.ndarray] = None,
```
Comment (AUdaltsova) [inline ocf_datapipes/transform/xarray/normalize.py:26]: I think the naming is a bit confusing given the `max_value` is already there and is quite different; also I'm pretty sure they are `xr.DataArray`-s?

File: ocf_datapipes/training/pvnet_site.py (hunk 1 above)
Comment (peterdudfield) [reply ocf_datapipes/training/pvnet_site.py:29]: I think `isort` orders them alphabetically

File: ocf_datapipes/transform/xarray/normalize.py (hunk 2 above)
Comment (peterdudfield) [reply ocf_datapipes/transform/xarray/normalize.py]: Good point

Comment (peterdudfield) [reply ocf_datapipes/transform/xarray/normalize.py]: Ive updated it now

File: ocf_datapipes/training/pvnet_site.py (hunk 1 above)
Comment (AUdaltsova) [reply ocf_datapipes/training/pvnet_site.py:29]: Ah! Shame. Sorry to nitpick for nothing :')

**Review by AUdaltsova — APPROVED** [review]

**Review by dfulu — APPROVED** [review]

**Pull request merged**